src/locales/en.json

```diff
@@ -324,33 +315,33 @@
       "instruction": "All details should be clear and readable"
     },
     "sms_failed": {
-      "message": "Something's gone wrong",
-      "instruction": "Copy the below link to your mobile instead"
+      "message": "Something’s gone wrong",
```

Comment:
Could you update the spreadsheet with a consistent apostrophe please? This one => `'`

Reply:
fixed